backend/src/middleware-callbacks/GroupUserCallbacks.ts

```diff
@@ -121,7 +121,7 @@ class GroupUserCallbacks {
                 else {
                   transporter.sendMail({
                     to: request.body.email,
-                    from: "257307@student.pwr.edu.pl",
```

Comment:
put some dummy email preferably, or put it as maybe env file and share it with us privately